src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -508,7 +508,7 @@ Use the dropdown to view and switch to other projects this file may belong to.</
     <value>Synchronizing with {0}...</value>
   </data>
   <data name="FullSolutionAnalysisOff" xml:space="preserve">
-    <value>Low memory detected. Full solution analysis disabled for this solution.</value>
+    <value>We have suspended some advanced features to improve performance.</value>
```

Comment:
@KevinH-MS that's what we wrote on the board at the meeting. adding joe. @morrisjoe